src/main/webapp/components/InternationalCohorts.css

```diff
@@ -9,17 +9,19 @@
     box-shadow:         0 1px 3px 1px #E1E1E1;
 }
 
-.questionContainer {
+.internationlCohortsBar .questionContainer {
     width: 323px;
     margin: 0 0 0 15px;
     padding-top: 20px;
 }
```

Comment:
Any time we touch a css file, we should be trying to extract code out of it instead of adding to it. This is a good example where a simple inline style object would suffice.

Reply:
I agree. We will try to do that in the following changes